### PR TITLE
Initial functionality.

### PR DESCRIPTION
**Pull request by humb1t**
Initial functionality:

- generation of keys via Ursa and `Ed25519`.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+use ursa::signatures::{ed25519::Ed25519Sha512, SignatureScheme};
+
+fn main() {
+    let (public_key, private_key) = Ed25519Sha512
+        .keypair(Option::None)
+        .expect("Failed to generate a ketpair.");
```
Comment (kortov) [inline src/main.rs]: “a ketpair” , a keypair maybe?

Comment (humb1t) [reply src/main.rs]: Thanks @kortov - will fix it.

**Comment by humb1t** [comment]
Ready to be merged into master.

File: src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    keys::{KeyGenOption, PrivateKey},
+    signatures::{ed25519::Ed25519Sha512, SignatureScheme},
+};
+
+/// This library contains utilities to work with cryptographical keys.
+
+/// If you need a keypair, use this method by default.
+/// It uses [Ursa](https://docs.rs/ursa) library with `Ed25519` solution of [EdDSA](https://tools.ietf.org/html/rfc8032).
+/// [OsRng](https://docs.rs/rand/0.7/rand/rngs/struct.OsRng.html) random number generator is used.
+pub fn generate_keypair() -> Result<(Vec<u8>, Vec<u8>), String> {
```
Comment (takemiyamakoto) [inline src/lib.rs:11]: If you know the length and it is immutable, why use Vec?

Comment (humb1t) [reply src/lib.rs:11]: By analogy with `ursa` API - will check if such a guarantees provided by their functions.

Comment (humb1t) [reply src/lib.rs:11]: Moved to an issue #2

**Comment by humb1t** [comment]
Commit signed-of

**Pull request merged**